1.basic/tasks.md

```diff
@@ -1,28 +1,48 @@
 #### write commands for following mongodb opertaions
 
-1. create a database named `sports`
-// Answer here ..
+- create a database named `sports`.
+  use sports
 
-2. list all databases present in local mongod server.
-// Answer here..
+- list all databases present in local mongod server.
+  show dbs
 
-3. create 3 collections named `cricket`, `football`, `TT` in sports database.
+- create 3 collections named `cricket`, `football`, `TT` in sports databse.
+  db.createCollection('cricket')
+  db.createCollection('football')
+  db.createCollection('TT')
 
-4. add 3 players in each of above collections which should have fields like `name`, `age`, `email`, state and auction_price.
+- add multiple players in those collections which should have fields like `name`, `age` and `email` and `cost`.
+  db.cricket.insert( {
+  ... name: "bim",
+  ... age : 24,
+  ... email: 'bim240@gmail.com',
+  ... cost: 2500000
+  ... }) and similar
 
-5. list all collections in sports database.
+* list all collections in sports database.
+  show dbs
```

Comment:
It should be ```show collections``` to list all collections from a connected database.
```js 
use sports
show collections
```

Reply:
done